factor/totp/classes/factor.php

```diff
@@ -184,7 +190,14 @@ public function setup_factor_form_definition_after_data($mform) {
         ]);
 
         $html = $togglelink . $html;
-        $mform->addElement('static', 'enter', '', $html);
+        $xssallowedelements[] = $mform->addElement('static', 'enter', '', $html);
+
+        // Allow XSS on Totara.
+        if (method_exists('MoodleQuickForm_static', 'set_allow_xss')) {
```

Comment:
Used an array (and already created elements) out of laziness to avoid having to deal with ordering.